source/lib-intro.tex

```diff
@@ -646,14 +646,29 @@
 
 \begin{itemize}
 \item
+Properties specified as locale-specific may change during program execution
```

Comment:
@jensmaurer: the paper italicizes "locale-specific" here; do we want to keep that?

Reply:
Yes, fixed.